docs/Guides/Migration-Guide-V5.md

```diff
@@ -0,0 +1,18 @@
+# V5 Migration Guide
+
+This guide is intended to help with migration from Fastify v4 to v5.
+
+Before migrating to v5, please ensure that you have fixed all deprecation
+warnings from v4. All v4 deprecations have been removed and they will no longer
+work after upgrading.
+
+## Breaking Changes
+
+### `useSemicolonDelimiter` false by default
+
+Starting with v5, every Fastify instance will not support the use of semicolon
+delimiter by default in querystring. The behavior in v4 and eariler versions is
+non standard behavior and this change should have no effect on the standard
+implementation of Fastify. You can revert this behavior by setting
+`useSemicolonDelimiter: true` in the server options.
```

Comment:
```suggestion
Beginning with version 5, Fastify instances will no longer utilize semicolons as delimiters when parsing querystrings by default. This decision was made due to the non-standard nature of semicolon delimiters, which are not supported by widely used querystring parsers like the built-in querystring package in Node.js.

However, if you still wish to use semicolons as delimiters, you can do so by setting the useSemicolonDelimiter option to true in the server configuration.
```

Reply:
Before you merge, @Fdawgs  and @jsumners you ok with the rephrasing?

Reply:
@Uzlopak i can take a look tomorrow

Reply:
@jsumners 
thanks bro :)